client/scss/layouts/_login.scss

```diff
@@ -9,17 +10,41 @@ $desktop-nice-padding: 100px;
 
 .login {
   margin-inline-start: 0;
-  height: 100%;
+  height: 100vh;
```

Comment:
@PaarthAgarwal FYI - some of these styles will be good to review in the context of you looking at the new auth pages redesigns. 

Also these changes will conflict with yours.